plutus-core/plutus-ir/src/PlutusIR/Purity.hs

```diff
@@ -5,10 +5,11 @@
 
 module PlutusIR.Purity (isPure, firstEffectfulTerm) where
 
+import PlutusCore.Builtin
+
 import PlutusIR
```

Comment:
It also breaks our policy of most "distant" modules going last.

Reply:
If we have such a policy I don't know it :sweat_smile: that's partly why I want to try and codfiy it!